go/mysql/endtoend/client_test.go

```diff
@@ -243,10 +243,10 @@ func expectNoError(t *testing.T, err error) {
 
 func expectFlag(t *testing.T, msg string, flag, want bool) {
 	t.Helper()
-	if flag != want {
+	require.Equal(t, want, flag, "%s: %v, want: %v",
 		// We cannot continue the test if flag is incorrect.
-		t.Fatalf("%s: %v, want: %v", msg, flag, want)
-	}
+		msg, flag, want)
+
```

Comment:
We should move the comment up to its own line IMO.

Reply:
(i'd argue this comment is superfluous; it's what `require` communicates IMO)